app/src/main/java/com/teamagam/gimelgimel/app/message/viewModel/adapter/MessagesRecyclerViewAdapter.java

```diff
@@ -143,7 +146,8 @@ private synchronized void animateSelection(MessageViewHolder viewHolder) {
         AnimatorSet set = new AnimatorSet();
         set.playSequentially(colorFade, reverseFade);
 
-        set.start();
+        viewHolder.setAnimatorSet(set);
+        viewHolder.startAnimation();
     }
 
     private void unselectCurrent() {
```

Comment:
GitHub won't let me comment on it because it's not a change you made, but I noticed that in two lines from here after calling to:
"mCurrentlySelected.setSelected(false);"
there should be an assignment:
"mCurrentlySelected = null;"
Currently it doesn't matter but I think we should fix this.